Add accessible labels for pricing tooltips (#1449)
primer/brand: .changeset/add-pricing-tooltip-label.md

---
'@primer/react-brand': patch
---

> Added `infoTooltipAriaLabel` to `PricingOptions.FeatureListItem` so consumers can distinguish info tooltip triggers when feature content uses rich React children.

Reviewer: "Rich" doesn't really make sense unless you already know about contentful.

```suggestion
Added `infoTooltipAriaLabel` to `PricingOptions.FeatureListItem` to customize the tooltip trigger's accessible label.
```

Author: Updated the changeset to use the suggested generic customization wording.
